### ISSUE TITLE
Styled components autocomplete doesn't work with typescript or js.

### ISSUE THREAD

**Issue by VolodymyrVoronov** [styled-components/vscode-styled-components#342]
Autocomplete doesn't work with styled-components' files, which are defined as a typescript or js files.

The plugins for vs-code were checked, reinstalled and installed a couple of times, but it didn't help.

See screenshots below.
1) No suggestions for display.
![image](https://user-images.githubusercontent.com/43247589/145683935-7cdcacda-23ad-46e2-8ea7-e24ba8a389c6.png)

2) No suggestions for font (mean: font-size oder font-family and so on).
![image](https://user-images.githubusercontent.com/43247589/145683983-71322413-52f8-4357-bc85-930f9a959345.png)

3) No suggestions for padding.
![image](https://user-images.githubusercontent.com/43247589/145684025-7a6ca99f-d667-48f3-95da-e4a6feadb988.png)

**Comment by VolodymyrVoronov** [comment]
Fixed with other plugin.

**Comment by JuanPastenCastillo** [comment]
How to you solve it @VolodymyrVoronov? Right now I have the same issue.

**Comment by VolodymyrVoronov** [comment]
> How to you solve it @VolodymyrVoronov? Right now I have the same issue.

Hi there @JuanPastenCastillo. 

I assume that there is new version of the plugin for vs code available. Please try to update or install the latest version of the plugin in your vs code. I hope it will solve the problem.

Best regards.

**Comment by JuanPastenCastillo** [comment]
Hi @VolodymyrVoronov. Thanks for your answer. I tried update the last version of the styled component but it not worked. Also I deactivated all the extensions and I start enabling one per one, but at the end nothing solve the problem. 

What I just did and it work, was the less elegant way to solve the problem: I copied and pasted all the project... and it worked. So, I don't change anything, I just copied everything.

One little thing, when I opened the original project in my vs code and the previous tab was shown in the screen, I can saw the little arrow/icon at the left to fold some piece of code for a moment as the image I attach.
![image](https://user-images.githubusercontent.com/46139914/230736706-0535eca6-94db-4fc7-bda5-02edb1f60437.png)
But just after charge some other things automatically in the vscode, I cant saw anymore those arrows/icons at the left
![image](https://user-images.githubusercontent.com/46139914/230737026-fadf31b2-e741-40fd-bf07-1c31c3e9c5c8.png)

Right now I'm using «"styled-components": "^5.3.9"» and «"next": "12.3.1"» on windows 10.
